.gitignore

```diff
@@ -1,11 +1,6 @@
 kubernetes/.packaged/
 .idea/
 *.swp
-.version
-
-# Binaries
-boot-script-service
```

Comment:
Why remove the binaries from the `.gitignore`?

Reply:
The binaries don't live there anymore and it was catching other files that should be part of the repo